### PR TITLE
Fix rewards breakdown on validator profile

### LINKED ISSUE #5068: "See breakdown" of rewards on validator profile doesn't work
### Expected behavior

There is a rewards tile where a user has the ability to view a breakdown of these rewards.

### Actual behavior

Clicking "See breakdown" does not work

image

### PR DESCRIPTION
**Pull request by ikem-legend**
### What was the problem?

This PR resolves #5068

### How was it solved?

- Updated the view based on [design](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=1450-206860&mode=design&t=weLPGTfD2om95jzE-0)

### How was it tested?

- Visually
- Unit tests

### CODE REVIEW

**Review by oskarleonard — DISMISSED** [review]

File: src/locales/en/common.json [hunk 1]
```diff
@@ -516,7 +516,7 @@
   "Review and sign": "Review and sign",
   "Reward": "Reward",
   "Reward amount": "Reward amount",
-  "Rewards": "Rewards",
+  "Rewards (LSK)": "Rewards (LSK)",
```
Comment (eniolam1000752) [inline src/locales/en/common.json]: Can rewards only be in LSK.

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js [hunk 2]
```diff
@@ -156,28 +161,50 @@ const PerformanceView = ({ data }) => {
...
-                <span>{t('Rewards')}</span>
-                <a>
-                  {t('See breakdown')}
-                  <Icon name="arrowBlueRight" />
-                </a>
+          <BoxContent className={`${styles.highlight} performance`}>
+            <div className={`${styles.content} ${styles.reward}`}>
+              <div className={`${styles.title} ${theme}`}>
+                <div>
+                  <span>{t('Rewards (LSK)')}</span>
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: (LSK) This shouldn't be hardcoded, use {tokenSymbol} for dynamic rendering

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js [hunk 3]
```diff
@@ -156,28 +161,50 @@ const PerformanceView = ({ data }) => {
...
+                  <div className={styles.separator} />
+                </div>
+                <div className={`${styles.highlightIcon}`}>
+                  <Icon name="reward" />
+                </div>
+              </div>
+              <div className={styles.details}>
+                <span>Self stake</span>
+                <Tooltip position="top" tooltipClassName={styles.tooltipDetails}>
+                  <span>Total rewards earned based on number of blocks generated.</span>
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: translation

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js [hunk 4]
```diff
@@ -156,28 +161,50 @@ const PerformanceView = ({ data }) => {
...
+                &nbsp;&nbsp;
+                <span className={styles.value}>
+                  <FormattedNumber val={Number(selfStakeRewardValue || 0).toFixed(4)} />
+                </span>
+              </div>
+              <div className={styles.details}>
+                <span>Commission</span>
+                <Tooltip position="top" tooltipClassName={styles.tooltipDetails}>
+                  <span>
+                    Total rewards earned by the validator as commission for generating blocks.
```
Comment (ManuGowda) [inline src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: same

**Review by ManuGowda — CHANGES_REQUESTED** [review]

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js (hunk 2 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: Fixed

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js (hunk 4 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: Fixed

File: src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js (hunk 3 above)
Comment (ikem-legend) [reply src/modules/pos/validator/components/ValidatorProfile/PerformanceView.js]: Fixed

**Review by ManuGowda — APPROVED** [review]

**Review by ManuGowda — CHANGES_REQUESTED** [review]
Screenshot 2023-06-23 at 11 44 31 AM

@ikem-legend when you make it full screen the tooltip is broken

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
## UI/UX issues observed

- The spacing between the texts is like 28px mean while figma says it should be 16.8px.
image

- The icon becomes smaller as the total rewards amount increases
image

File: src/locales/en/common.json (hunk 1 above)
Comment (ikem-legend) [reply src/locales/en/common.json]: Rewards are in the POS token

**Review by ManuGowda — APPROVED** [review]

**Review by eniolam1000752 — APPROVED** [review]

**Pull request merged**